packages/nimble-components/src/rich-text/components/mention-view/mention-users-view/tests/rich-text-mention-users-view-matrix.stories.ts

```diff
@@ -0,0 +1,69 @@
+import { ViewTemplate, html } from '@microsoft/fast-element';
+import type { Meta, StoryFn } from '@storybook/html';
+import {
+    createMatrix,
+    sharedMatrixParameters
+} from '../../../../../utilities/tests/matrix';
+import { createMatrixThemeStory } from '../../../../../utilities/tests/storybook';
+import { richTextMentionUsersViewTag } from '..';
+import {
+    disabledStates,
+    type DisabledState
+} from '../../../../../utilities/tests/states';
+import {
+    bodyFont,
+    bodyFontColor,
+    borderColor,
+    borderWidth,
+    mediumPadding,
+    smallPadding
+} from '../../../../../theme-provider/design-tokens';
+
+const metadata: Meta = {
+    title: 'Tests/Rich Text Mention: User',
+    parameters: {
+        ...sharedMatrixParameters()
+    }
+};
+
+export default metadata;
+
+const component = ([
+    disabledName,
+    disabled
+]: DisabledState): ViewTemplate => html`
+    <style class='code-hide'>
+        .mention-container {
```

Comment:
Let's not set font styles on something that contains the mention element. That makes it ambiguous whether the test is validating the default appearance of the element or the appearance caused by custom storybook styles.

If the goal is to style the descriptive text next to the element you could put that in a `<span>` and target the span. Or you could follow my other suggestion and use the mention-label for the descriptive text and delete this styling.

Reply:
Yeah, makes sense. Since we don't want the other string to be in the mention-label, I just wrapped it in span, added styles, and removed it from the container class.